design/object-bucket/object-bucket.md

```diff
@@ -3,57 +3,47 @@
 ## Proposed Feature
 
 Enable Rook-Ceph Object users the ability to request buckets via an ObjectBucketClaim, similar
-to requesting a Persistent Volume using a Persistent Volume Claim. 
+to requesting a Persistent Volume using a Persistent Volume Claim.
 
 ### Overview
 
-There is recent support for a bucket provisioning library, very similar in concept to the 
-[sig-storage-lib-external-provisioner](https://github.com/kubernetes-sigs/sig-storage-lib-external-provisioner/blob/master/controller/controller.go) library. See the
-[lib-bucket-provisioner](https://github.com/yard-turkey/lib-bucket-provisioner) repo for more
-details. The proposal here covers only the design of Rook-Ceph RGW specific bucket provisioning.
-
-Rook-Ceph has developed Custom Resource Definitions (CRDs) to support the dynamic provisioning of object stores and users (`CephObjectStore`s and `CephObjectStoreUser`s, respectively).  This proposal introduces the next logical step in this design: the addition a controller for handling dynamic bucket creation requests leveraging the ObjectBucketClaim and ObjectBucket library.  The intent is to round out the Rook-Ceph experience for Kubernetes users by providing a single control plane for the managing of Ceph Object components.  The bucket provisioning lib does all of the heavy lifting in terms of watching OBCs and responding to events. The new Rook-Ceph code will support `Provision()` and `Delete` methods and adhere to the generic bucket provisioning interface.
+Currently, the Rook-Ceph operator enables the dynamic creation of Ceph cluster, object stores, and users via Rook native Custom Resource Definitions.  This proposal introduces the next logical step in this design: the addition a controller for handling dynamic bucket creation by leveraging the [lib-bucket-provisioner library](https://github.com/yard-turkey/lib-bucket-provisioner).  This library defines two new CRDs: ObjectBucketClaim and ObjectBucket.  Additionally, the library wraps a Kubernetes to watch related API events.
```

Comment:
nit: "Ceph clusters..." plural

Reply:
_Additionally, the library wraps a Kubernetes to watch related API events._
Additionally, the library provides, to all bucket provisioners, a Kubernetes controller to watch related API events.

Reply:
comment still applies. "cluster" should be plural or write "a cluster..."